packages/api-client/templates/modular/http-client.ejs

```diff
@@ -34,6 +34,10 @@ export interface ApiConfig<SecurityDataType = unknown> {
     customFetch?: typeof fetch;
 }
 
+export type ThrowableResponse<T,E> = Promise<HttpResponse<T,E>> & {
+    throwOnError: () => Promise<T>
```

Comment:
Please add documentation for this function so it will be a hint for `api-client` users